### PR TITLE
build(editor): bundle tailwind css output with js

### PR DESCRIPTION
**Pull request by hejtful**
Changes:
- Exported CSS now bundled with the editor JS, meaning one less thing to manually import in integrations

Drawback:
- Using Serlo CSS is no longer optional.

Implementation example in: https://github.com/serlo/serlo-editor-for-edusharing/pull/351

### CODE REVIEW

**Review by CodingDive — APPROVED** [review]
Thank you! 

> Using Serlo CSS is no longer optional.

But can be overwritten by editor consumers, right?

**Comment by hejtful** [comment]
> But can be overwritten by editor consumers, right?

Yes, not in the nicest way, but this PR didn't introduce any regressions there.

File: packages/editor/package.json [hunk 1]
```diff
@@ -19,6 +19,10 @@
     "./dist/editor-tailwind.css": {
       "import": "./dist/editor-tailwind.css",
       "require": "./dist/editor-tailwind.css"
+    },
+    "./dist/style.css": {
+      "import": "./dist/style.css",
+      "require": "./dist/style.css"
```
Comment (hejtful) [inline packages/editor/package.json:25]: Note for myself: Check if this is actually necessary

**Review by LarsTheGlidingSquirrel — APPROVED** [review]
Looks good! Hopefully this will make it easier in the future.

**Pull request merged**